### PR TITLE
loggerd: implement DONT_LOG flag

### PR DESCRIPTION
**Pull request by geohot**
Some params have sensitive information. Don't put them in the logs.

### CODE REVIEW

File: selfdrive/loggerd/logger.cc [hunk 1]
```diff
@@ -108,7 +108,9 @@ kj::Array<capnp::word> logger_build_init_data() {
   for (auto& [key, value] : params_map) {
     auto lentry = lparams[i];
     lentry.setKey(key);
-    lentry.setValue(capnp::Data::Reader((const kj::byte*)value.data(), value.size()));
+    if ( !(params.getKeyType(key) & DONT_LOG) ) {
```
Comment (adeebshihadeh) [inline selfdrive/loggerd/logger.cc:111]: hmm, are you sure the precedence is right?

Comment (adeebshihadeh) [reply selfdrive/loggerd/logger.cc:111]: might be better to add a testKey helper

Comment (geohot) [reply selfdrive/loggerd/logger.cc:111]: Not sure what you mean. Is that wrong somehow?

`batman@x840:/tmp$ cat a.c && ./a.out
#define DONT_LOG 0x40
int main() {
  int a = 0x40;
  int b = 0;
  if ( !(a & DONT_LOG) ) printf("not set on a\n");
  if ( !(b & DONT_LOG) ) printf("not set on b\n");
}
not set on b
`

Comment (adeebshihadeh) [reply selfdrive/loggerd/logger.cc:111]: missed the parantheses

**Pull request merged**